### DISCUSSION TITLE
PrimeNG Sakai - Font error issue

### DISCUSSION THREAD

**Discussion by DanPureCMMS** [primefaces/community#1032] [PrimeNG Templates]
Hi everyone,
I've initiated a new PrimeNg project and I choose to use Sakai.
Everything seems ok at the exception of this error which is raised despite the version of Angular (16 or 17), see below:
`ERROR RuntimeError: NG04002: Cannot match any routes. URL Segment: 'assets/layout/styles/theme/lara-light-indigo/fonts/Inter-roman.var.woff2' `

**Comment by cetincakiroglu** [comment]
Hi,

Could you please tell us the steps you've done to replicate the behavior?

**Comment by DanPureCMMS** [comment]
Hello
In fact, I found out that the library was not in the expected, so I had to download it then to store it properly
